drivers/usb/device/usb_dc_stm32.c

```diff
@@ -258,14 +258,14 @@ static int usb_dc_stm32_clock_enable(void)
 #ifdef CONFIG_USB_HS_BASE_ADDRESS
 
 
-#ifdef USB_HS_PHYC
+#ifdef CONFIG_USBPHYC_BASE_ADDRESS
```

Comment:
We're now generating CONFIG_DT_COMPAT_XXXX flags (more details here: #9406).
These should be used for driver specific implementations/variations. Since you've set the ground with phys nodes it would fit perfectly here.
Please use this instead of 'CONFIG_USBPHYC_BASE_ADDRESS'.
Apply everywhere.

Reply:
I changed it to DT_COMPAT_ST_STM32_USBPHYC.